modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/list-item-with-icon/index.html

```diff
@@ -0,0 +1,17 @@
+<li class="fragment_71 d-flex justify-content-center flex-column align-items-center">
+	<div class="align-items-center d-flex justify-content-between main-div py-2">
+		<div class="mt-3">
+			<p data-lfr-editable-id="main-text" data-lfr-editable-type="text">
+				Frequently asked question about something related to insurance?
+			</p>
+		</div>
+			<img alt="Plus icon" class="custom-icon" id="plus-icon" src="[resources:plus.svg]" />
+
+			<img alt="Minus icon" class="custom-icon d-none" id="minus-icon" src="[resources:minus.svg]" />
+	</div>
+
+	<div class="collapse main-div" id="collapseDiv">
```

Comment:
```suggestion
	<div class="collapse main-div" id="collapse-div">
```

Reply:
Also, you don't should use the name of element as **id** or **class**, sounds a little bit redundant.

```suggestion
	<div class="collapse faq-questions" id="faq-list">
```

Something like this.

Reply:
I'll try to be more semantic while creating class names. Thank you.